src/com/reason/ide/files/DuneFileType.java

```diff
@@ -1,25 +1,32 @@
 package com.reason.ide.files;
 
+import com.google.common.collect.ImmutableSet;
 import com.intellij.openapi.fileTypes.FileType;
 import com.intellij.openapi.fileTypes.LanguageFileType;
-import icons.ORIcons;
 import com.reason.lang.dune.DuneLanguage;
+import icons.ORIcons;
 import org.jetbrains.annotations.NotNull;
 import org.jetbrains.annotations.Nullable;
 
 import javax.swing.*;
+import java.util.Set;
 
 public class DuneFileType extends LanguageFileType {
+
     public static final FileType INSTANCE = new DuneFileType();
 
+    public static Set<String> getDefaultFilenames() {
+        return ImmutableSet.of("dune", "dune-project", "jbuild");
+    }
+
     private DuneFileType() {
         super(DuneLanguage.INSTANCE);
     }
 
     @NotNull
     @Override
     public String getName() {
-        return "Dune configuration";
+        return "Dune Configuration";
```

Comment:
there's an annotation on the base class that wants the description to be sentence-cased so I'm assuming the name should be title-cased